classes/Language.php

```diff
@@ -1219,33 +1225,20 @@ public static function downloadLanguagePack($iso, $version, &$errors = [])
             return false;
         }
 
-        return static::downloadXLFLanguagePack($lang_pack['locale'], $errors, self::PACK_TYPE_SYMFONY);
+        return static::downloadXLFLanguagePack($lang_pack['locale'], $errors);
     }
 
     /**
      * Downloads a language pack into local cache
      *
      * @param string $locale IETF language tag
      * @param array $errors
-     * @param string $type self:PACK_TYPE_SYMFONY|self::PACK_TYPE_EMAILS
      *
      * @return bool
      */
-    public static function downloadXLFLanguagePack($locale, &$errors = [], $type = self::PACK_TYPE_SYMFONY)
+    public static function downloadXLFLanguagePack($locale, &$errors = [])
```

Comment:
Please deprecate the `$type` parameter rather than removing it

https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations